builder/antlr/GrammarAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package com.vaticle.dependencies.builder.antlr
+
+
+import picocli.CommandLine
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.io.File
+import java.util.concurrent.Callable
+import kotlin.system.exitProcess
+
+
+@Command(name = "grammar-adapter", mixinStandardHelpOptions = true)
+class GrammarAdapter : Callable<Unit> {
+
+    @Option(names = ["--in"], required = true)
+    lateinit var inputFile: File
+
+    @Option(names = ["--out"], required = true)
+    lateinit var outputFile: File
+
+    @Option(names = ["--keyword"], required = true)
```

Comment:
Can we change it to `--adapt-keyword`?

When put together with the other option, `--keyword` is not clear:

```
--input in.g4 --output  out.g4 --keyword type --keyword filter
```

this is clearer

```
--input in.g4 --output  out.g4 --adapt-keyword type --adapt-keyword filter
```

Reply:
@lolski addressed in https://github.com/vaticle/dependencies/pull/283/commits/594fd64558c895bae69d7b57cec6ef0d9d8f15b6